packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/QuickActionsPluginTests.swift

```diff
@@ -143,28 +90,28 @@ class QuickActionsPluginTests: XCTestCase {
       userInfo: nil)
 
     let invokeMethodExpectation = expectation(description: "invokeMethod must be called.")
-    mockChannel.invokeMethodStub = { method, arguments in
-      XCTAssertEqual(method, "launch")
-      XCTAssertEqual(arguments as? String, item.type)
```

Comment:
Here and later, it seems like we're losing important test coverage; shouldn't we still be asserting that the right type is provided?